app/enterprise/2.1.x/kong-manager/oidc-mapping.md

```diff
@@ -0,0 +1,258 @@
+Basic Auth and are transitioning to OpenID Connect.
+</div>
+
+{% navtabs %}
+{% navtab Kong Manager %}
+
+Create a group and assign a role to it:
+
+1. Open **Teams** from the top navigation.
+2. Click the **Groups** tab.
+3. Click **Create New Group**.
+4. Set the **Group Name** to match your IdP group.
+
+    **Note:** Group names are case-sensitive. Make sure to match your IdP
+    group name exactly.
+
+5. (Optional) In the **Comment** field, enter a description for the group.
+6. Click on **Add/Edit Roles**, then choose a workspace and a role.
+7. Save the role assignment, then click **Create**.
+
+Create an admin for the group:
+1. Open the **Teams** from the top navigation.
+2. Click the **Admins** tab.
+3. Click **Invite Admin**.
+4. Enter a username and email, and optionally a custom ID to make the admin
+easy for you to identify.
+
+    **Note**: Make sure the username exactly matches the admin's name in
+    your IdP.
+
+5. Ensure the **Enable RBAC token** checkbox is checked.
+6. Save the role assignment, then click **Invite Admin**.
+
+{% endnavtab %}
+{% navtab Kong Admin API %}
+
+1. Create a group, making sure the group `name` parameter matches your IdP group
+name:
+
+    ```sh
+    $ curl -X POST --url http://<admin-hostname>:8001/groups \
+      --header 'content-type: application/json' \
+      --header 'kong-admin-token: <yourtoken>' \
+      --data '{
+          "comment": "example group",      
+          "name": "examplegroup"      
+        }'
+    ```
+
+    **Note:** Group names are case-sensitive. Make sure to match your IdP group
+    name exactly.
+
+2. Assign a role to the group:
+
+    ```sh
+    $ curl -X POST --url http://<admin-hostname>:8001/groups/{group-id}/roles \
+      --header 'content-type: application/json' \
+      --header 'kong-admin-token: <yourtoken>' \
+      --data '{   
+          "rbac_role_id": "e948171e-699c-4035-9b74-2b2b576d9644",
+    	    "workspace_id": "236bfa99-cf09-4389-afa8-e2bd6da89fd3"
+        }'
+    ```
+
+    Where:
+    * [`rbac_role_id`](/enterprise/{{page.kong_version}}/admin-api/rbac/reference/#list-roles):
+    UUID of the role you want to assign.
+    * [`workspace_id`](/enterprise/{{page.kong_version}}/admin-api/workspaces/reference/#list-workspaces):
+    UUID of the workspace to add the role to.
+
+3. Create an admin for the group:
+
+    ```sh
+    $ curl -X POST --url http://<admin-hostname>:8001/admins \
+      --header 'content-type: application/json' \
+      --header 'kong-admin-token: <yourtoken>' \
+      --data '{
+      "username": "<someusername>",
+      "custom_id": "<examplename>",
+      "email": "<your-email@company.com>",
+      "rbac_token_enabled": true
+    }'
+    ```
+
+{% endnavtab %}
+{% endnavtabs %}
+
+Notice how in the instructions above, you did not assign a role to your
+admin. The role will be matched with the role assigned to them in the IdP.
+
+## Apply OIDC Auth Mapping to Kong
```

Comment:
Might be worth to include a table of the required params for mapping, if there's more than just `authenticated_groups_claim`.